From ee13d45d8ce6f7711eedfedeb0d6d6750b524d27 Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Mon, 8 Sep 2014 15:01:01 +0900
Subject: [PATCH] oneirick does not have python2

---
 Makefile.openrtm_aist_python |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.openrtm_aist_python b/Makefile.openrtm_aist_python
index e47069e..d9fcd9f 100644
--- a/Makefile.openrtm_aist_python
+++ b/Makefile.openrtm_aist_python
@@ -11,7 +11,7 @@ MD5SUM_FILE = $(FILENAME).md5sum
 MK_DIR      = $(shell rospack find mk)
 
 include $(MK_DIR)/download_unpack_build.mk
-PYTHONBIN   = python2
+PYTHONBIN   = python
 PYTHONDIR   = `$(PYTHONBIN) -c 'import sys; print "python%s"%sys.version[:3]'`
 SETUPTOOLS_ARG_EXTRA = "--install-layout=deb"
 
-- 
1.7.9.5

